progenitor-impl/src/method.rs

```diff
@@ -1168,7 +1168,6 @@ impl Generator {
 
         // TODO to deal with multiple response types, we'll need to create an
         // enum type with variants for each of the response types.
-        assert!(response_types.len() <= 1);
```

Comment:
This fixes https://github.com/oxidecomputer/progenitor/issues/344

I can remove it by adding a lot more fiddling to `test_github`